.devcontainer/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM debian:10
+
+# This Dockerfile adds a non-root 'vscode' user with sudo access. However, for Linux,
+# this user's GID/UID must match your local user UID/GID to avoid permission issues
+# with bind mounts. Update USER_UID / USER_GID if yours is not 1000. See
+# https://aka.ms/vscode-remote/containers/non-root-user for details.
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# Set default locale for the environment
+ENV LC_ALL C.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+
+EXPOSE 4000
+
+# Configure apt and install packages
+RUN apt-get update \
+    && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends apt-utils dialog 2>&1 \
+    #
+    # Install vim, git, process tools, lsb-release
+    && apt-get install -y \
+        locales \
+        git \
+        openssh-client \
+        less \
+    #
+    # Install ruby
+    && apt-get install -y \
```

Comment:
Is there any technical need to split `apt-get` installation transactions? Unless it doesn't work otherwise, I suggest to merge all packages to a single installation batch.

Reply:
That separation comes from the original Dockerfile template generated by VS Code. 
I think it is mostly logical as there should be no technical reason for them to be in separate batches.
I'm gonna try to merge them.

Reply:
@zvirja I have merged the install instructions